src/components/ResumeContent.tsx

```
@@ -783,21 +1125,125 @@
+                            <path d="M18 8.118l-8 4-8-4V14a2 2 0 002 2h12a2 2 0 002-2V8.118z" />
+                          </svg>
+                          <span className="text-gray-600 dark:text-gray-400 text-sm">
+                            {ref.email}
+                          </span>
+                        </div>
+                      )}
+
+                      {ref.reference && (
+                        <div className="border-t border-gray-200 dark:border-gray-700 pt-4">
+                          <p className="text-gray-700 dark:text-gray-300 text-sm leading-relaxed italic">
+                            "{ref.reference}"
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)